CMakeConfig.txt

```diff
@@ -80,6 +80,7 @@ set(PROTOCOL_SUNRISESET ON CACHE BOOL "support for the Sunrise / Sunset calculat
 set(PROTOCOL_TECHLICO ON CACHE BOOL "support for TechLiCo Lamps protocol")
 set(PROTOCOL_TEKNIHALL ON CACHE BOOL "support for the Teknihall weather station protocol")
 set(PROTOCOL_TFA ON CACHE BOOL "support for the TFA weather station protocol")
+set(PROTOCOL_UNITEC ON CACHE BOOL "support for the unitec switch protocol")
```

Comment:
I can follow your argumentation and even like being precise; but then a lot of protocol names should be changed to be more specific just in case a different kind of device comes under the same brand.

For symmetry of protocol names in pilight I would prefer to wait until one implements the unitec push button and founds out that is really a different protocol.